### DISCUSSION TITLE
No modbus communication after update to 4.22.44

### DISCUSSION THREAD

**Discussion by andreas0maurer** [WillCodeForCats/solaredge-modbus-multi#719] [General]
Does anybody see the same experiance? I have update two of my four WR to 4.22.44 yesterday. These two can not be polled from the home assistant integration anymore.
evcc can still get the requested info. Looks like your integration will fail on the answers. 
Any advice.

**Comment by cyberkryten** [comment]
I didn't notice any modbus issues when the system upgraded to 4.22.44, but you might have issues if only part of the system has been upgraded.  

It is also possible something during the upgrade disabled Modbus TCP so you may have to to back into SetApp and re-enable it.
